packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/OVEAssetModel/Create.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@model OVE.Service.AssetManager.Models.OVEAssetModel
+@using OVE.Service.AssetManager.Domain
+@inject ServiceRepository FileTypes
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h2>Create</h2>
+
+<h4>ImageFileModel</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Create" enctype="multipart/form-data">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="Project" class="control-label"></label>
+                <input asp-for="Project" class="form-control" />
+                <span asp-validation-for="Project" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Name" class="control-label"></label>
+                <input asp-for="Name" class="form-control" />
+                <span asp-validation-for="Name" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Description" class="control-label"></label>
+                <input asp-for="Description" class="form-control" />
+                <span asp-validation-for="Description" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Service" class="control-label"></label>
+                @Html.DropDownList("Service", FileTypes.GetServices(),new { @class = "form-control" })
+                <span asp-validation-for="Service" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="AssetMeta" class="control-label"></label>

```

Comment:
What is AssetMeta and how does it differ from Description?
I believe some more detail is required for these - I would recommend that all of the fields have an Input placeholder with a description of what is required

Reply:
AssetMeta is an optional place to store optional meta data about the asset which can be updated through API by other services. 

Have documented this.